docs/self-managed/identity/orchestration-identity/installation.md

```diff
@@ -0,0 +1,53 @@
+---
+id: installation
+title: Installation
+description: "Learn how Identity is bundled with your default Orchestration cluster."
+---
+
+Identity is included by default with the deployment of any [Orchestration cluster](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster). Within an Orchestration cluster, Identity provides unified, cluster-level identity management and authorizations.
+
+Identity for Orchestration clusters is available via [Helm install](/self-managed/setup/install.md), and for local development via [Camunda 8 Run](/self-managed/setup/deploy/local/c8run.md).
+
+## Initial configuration
+
+Following the default installation will result in a cluster with the following:
+
+1. Basic authentication enabled
+2. API authentication disabled
+3. Authorizations disabled
+4. An initial user with the username/password `demo`/`demo`
+5. An `admin` role with read, create, update, and delete permissions for other roles
```

Comment:
@Ben-Sheppard I've added your suggestions here, but please let me know if this is correct. :) We don't have any docs on roles yet, I believe, so linking this to other content will have to come in a future update.

Reply:
Hmm maybe this is getting to my question above - we have an initial user and then an admin role. Is the admin role applied to that initial user?

Reply:
Thats correct @akeller - the admin role is applied to the initial user to support a better getting started experience